1-Pflichtenheft/pflichtenheft.tex

```diff
@@ -88,6 +88,99 @@ \section{Produktumgebung}
+			\item Den Kandidat loeschen Falls diese Option ausgewählt ist, gehen Sie zu 5.7)
+		\end{enumerate}
+		\item Falls die Wahlen schon abgeschlossen sind: Das System bietet dem Administrator die Wahl nur eine  der   folgenden Möglichkeiten:
+		\begin{enumerate}
+			\item Berechnen die aktuellen Anzahl der Votes. (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.3)
+			\item Neue Wahlen beginnen (Falls Sie diesen Punkt auswählen, gehen Sie zu 5.2.2)
+		\end{enumerate}
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Zaehlen der Votes}	
+\begin{itemize}
+	\item Ziel: berechnen die Anzahl der von Waehler gegebenen Votes.
+	\item Verwaltung der Wahlen.
+	\item Das System gibt Informationen über jeden Kandidaten (Name, Nachname, Foto des Kandidaten) und die Anzahl der von jedem der Wähler erhaltenen Votes aus. Der Netzwerkadministrator nach der Überprüfung der Informationen geht zurück (siehe 5.2.1 oder 5.2.3). 
+\end{itemize}
+
+\subsection{Beendigung der Wahlen}
+\begin{itemize}
+	\item Ziel: die Wahlen beenden.
+	\item Subsystem: Verwaltung der Wahlen.
+      \item Das System gibt die Absichten des Netzwerkadministrators an, den Abschluss der Wahlen durchzuführen.
+Der Administrator stimmt zu oder lehnt ab.
+	\begin{enumerate}
+		\item Falls Netzwerkadministrator stimmt zu: Das System stoppt den Wahlenprozess. (Dann Uebergang 
+zu 5.2)
+		\item Falls Netzwerkadministrator lehnt ab: Uebergang zu 5.2
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Hinzufuegen des neuen Kandidaten}
+\begin{itemize}
+	\item Ziel: Neue Kandidat in dem Wahlprozess hinzufuegen.
+	\item Subsystem: Verwaltung der Wahlen.
+	\item Das System schlägt vor, Nachname, Vorname, Vatersnamen einzugeben und ein Foto des Kandidaten hochzuladen.
+	\item Der Administrator gibt den Nachnamen, den Namen und den Vatersnamen ein und lädt das Foto des Kandidaten.
+	\item Das System prüft die Daten auf Korrektheit.
+	\begin{enumerate} 
+		\item Wenn die Daten nicht korrekt sind, informiert das System den Benutzer darüber und wechselt zu 5.2.
+		\item Wenn die Daten korrekt sind, fordert das System eine Bestätigung an und fügt, falls bestätigt, der Datenbank Informationen über den neuen Kandidaten hinzu. 
+	\end{enumerate}
+	\item Dann Uebergang zu 5.2.
+\end{itemize}
+
+\subsection{Erstellung der Wahlen}
+\begin{itemize}
+	\item Ziel: “initialize” der Wahlen.
+	\item Verwaltung der Wahlen.
+	\item Das System gibt die Absichten des Netzwerkadministrators an, den Abschluss der Wahlen durchzuführen.
+Der Administrator stimmt zu oder lehnt ab.
+	\begin{enumerate}
+		\item Falls Netzwerkadministrator stimmt zu: Das System stoppt den Wahlenprozess. (Dann Uebergang 
+zu 5.2)
+		\item Falls Netzwerkadministrator lehnt ab: Uebergang zu 5.2
+	\end{enumerate}
+\end{itemize}
+
+\subsection{Loeschung der Kandidaten}
+\begin{itemize}
+	\item Ziel: einen Kandidaten aus dem Wahlenprozess entfernen (Z.B, falls der Kandidat sich vor Beginn der Wahlen geweigert hat teilzunehmen).
```

Comment:
@Schaback @tinytiny123 @achimkr was denkt ihr dazu, räumt das einem Wahlleiter nicht eventuell zu viele Rechte ein? wenn ein Wähler nicht wählen möchte, dann gibt er seine Stimme ja annahmsweise sowieso nicht ab, also besteht auch kein Grund ihn aus den Wahlen zu entfernen.
Was ist allerdings in dem Fall, dass ein Wähler gewählt hat: der Administrator könnte ihn dann trotzdem "löschen", eventuell führt das zu eigenartigem verhalten je nachdem wie wir das implementieren würden?

Reply:
Ich denke wir sollten uns generell mal überlegen wann was geschieht. Meiner Meinung nach sollte der Wahlleiter eine Wahl aufsetzen können, dann die Kandidaten hinzufügen und dann die Wahl starten also die Stimmabgabe freischalten. Insofern denke ich könnte man das nach dem aufsetzen der Wahl noch vor Freischaltung der Stimmabgabe erlauben. Nach dem starten der Wahl würde ich das nichtmehr erlauben.